crates/ruff_python_formatter/src/other/comprehension.rs

```diff
@@ -56,7 +56,7 @@ impl FormatNodeRule<Comprehension> for FormatComprehension {
             [
                 token("for"),
                 trailing_comments(before_target_comments),
-                group(&format_args!(
+                format_args!(
```

Comment:
The `format_args` here isn't necessary. You can pass all elements directly to `write`
```suggestion
```

Sorry, GitHub doesn't allow me to make the suggestion :(